CMakeLists.txt

```diff
@@ -245,7 +249,8 @@ else() # Standalone build
 		RUNTIME_OUTPUT_DIRECTORY_DEBUG Debug
 		RUNTIME_OUTPUT_DIRECTORY_RELEASE Release)
 	target_include_directories(qasm_simulator PRIVATE ${AER_SIMULATOR_CPP_SRC_DIR})
```

Comment:
@hhorii this is not your fault (is mine actually), but I find out that we need to fix this line to something like:
```
target_include_directories(qasm_simulator
		PRIVATE ${AER_SIMULATOR_CPP_SRC_DIR}
		PRIVATE ${AER_SIMULATOR_CPP_EXTERNAL_LIBS})
```
Could you change this as well? Thanks!